respa_admin/views/resources.py

```diff
@@ -61,6 +64,12 @@ def get_queryset(self):
             qs = qs.filter(type=self.resource_type)
         if self.resource_unit:
             qs = qs.filter(unit=self.resource_unit)
+        if self.order_by:
```

Comment:
There could be a simple test case which runs the order_by code here. Currently it's not covered by any test.